proto/buf/alpha/module/v1alpha1/module.proto

```diff
@@ -65,6 +65,10 @@ message Module {
   //
   // string is used to enforce UTF-8 encoding or 7-bit ASCII text.
   string license = 6;
+  // documentation_filename is the filename of the file which contains the module documentation.
+  //
+  // either `buf.md`, `README.md` or `README.markdown`.
+  string documentation_filename = 7;
```

Comment:
I would call this `documentation_path` in case in the future we want to support this being in another directory

Reply:
Makes sense. I'll get that changed. Thanks!